tests/clipboard.spec.ts

```diff
@@ -176,10 +176,10 @@ test('split block when paste', async ({ page }) => {
   await page.keyboard.type('abc');
   await setQuillSelection(page, 1, 1);
   await pasteContent(page, clipData);
-  await assertRichTexts(page, ['abtext', 'h1c']);
+  await assertRichTexts(page, ['atext', 'h1', 'c']);
   await assertSelection(page, 1, 2, 0);
   await undoByClick(page);
-  await assertRichTexts(page, ['\n']);
```

Comment:
why it's ac?

Reply:
When I use single step debugging in vscode, I find that this step is a retraction, so the result should be 'ac'

Reply:
To split the words, I used the ready-made `handleBlockSplit` method, which splits the block of text into two pieces

Reply:
I'll upload a video to do the same operation